fuzzers/fuzzbench_fork_qemu/Cargo.toml

```diff
@@ -13,6 +13,6 @@ debug = true
 
 [dependencies]
 libafl = { path = "../../libafl/" }
-libafl_qemu = { path = "../../libafl_qemu/", features = ["x86_64"] }
+libafl_qemu = { path = "../../libafl_qemu/", features = ["x86_64", "usermode"] }
```

Comment:
Maybe just `system` and `user` are enough?

Reply:
Ah Usermode has already been called like that

Reply:
Exactly. I think the code is more readable if we write
```cfg(emulation_mode = "systemmode")```
instead of
```cfg(not(feature = "usermode"))```

We could maybe think about using "user" and "system" instead of "usermode" and "systemmode" tho.